community/releases/versions.md

```diff
@@ -5,6 +5,22 @@ sidebar:
   nav: community
 ---
 
+The Spinnaker releases listed below are top-level versions tying together each
+Spinnaker subcomponents' versions. These have been validated together in an 
+end-to-end integration test suite curated by the Spinnaker community, and
+represent a more maintainable, easy to upgrade Spinnaker. While 
+it's still possible to install each Spinnaker subcomponent's versions 
```

Comment:
The point is for this to contrast the existing install path (downloading the packages via whatever path users are going by) against the Halyard install path. It's not explaining that Halyard can allow you to download whatever versions you want.